src/Component/Footer/Footer.jsx

```diff
@@ -1,44 +1,81 @@
 import React from 'react';
+import ReactDOM from 'react-dom';
 
 import "./footcss.css";
-import inslogo from "./inslogo.png";
-import dotlogo from "./dot.png";
-import cclogo from "./inslogo.png";
+import 'font-awesome/css/font-awesome.min.css';
+
+//import images
+import inslogo from "../../Assets/Images/inslogo.png";
+import dotlogo from "../../Assets/Images/dot.png";
+import cclogo from "../../Assets/Images/inslogo.png";
+
 
 const Footer = () => {
+ 
+  // var toogle = true;
+  const link_toogle = () => {
+    var element = document.getElementById("cyka");
+
+  if (element.classList) { 
+    element.classList.toggle("link-display");
+  } else {
+    var classes = element.className.split(" ");
+    var i = classes.indexOf("link-display");
+
+    if (i >= 0) 
+      classes.splice(i, 1);
+    else 
+      classes.push("link-display");
+      element.className = classes.join(" "); 
+  }
+
+    // let element = document.getElementById('cyka')
+    // ReactDOM.findDOMNode(element).style.display='block'    
+
+    // toogle =  toogle? false : true ;
+    // console.log(toogle);
+  };
+
   return (
     <footer>
       <div className="inname">
         <div>
             <img src = {inslogo} alt="iiitv-logo" className="inslog"></img>
         </div>
-        <div className="name">
+        <div className="iiitv-name">
           Indian Institute of Information Technology Vadodara
         </div>
       </div>
 
       <div className="linkcon">
+
         <div className="footerlin">
-        <div className="linkna">
-          LINKS
-          <hr className="underline"></hr>
-        </div>
-        <div className="links">
-          <a href="" className="link">Home</a>
-          <a href="" className="link">Events</a>
-          <a href="" className="link">Members</a>
-          <a href="" className="link">Institute Site</a>
-        </div>
+          <div className="link-drop">
+            <div className="linkna">
+              LINKS
+              <hr className="underline"></hr>
+            </div>
+            <div class="link-toogle" onClick = {() => link_toogle()} >
```

Comment:
a piece of advise there is also something called bind function which can be an alternative here

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/bind